website/src/docs/hotchocolate/filtering/filter-customization.md

```diff
@@ -0,0 +1,424 @@
+# Customizing Filter
```

Comment:
Frontmatter missing...

```
---
title: "Here goes the headline..."
---
```

And do not use single `#` for subtitles; start with `##`.